src/content/docs/extensions/attach/postgres.mdx

```diff
@@ -0,0 +1,279 @@
+| smallint (int2)                           | INT16                           |
+| smallserial (serial2)                     | INT16                                 |
+| serial (serial4)                          | INT32                                 |
+| text                                      | STRING                          |
+| time [ (p) ] [ without time zone ]        | UNSUPPORTED                            |
+| time [ (p) ] with time zone (timetz)      | UNSUPPORTED                                 |
+| timestamp [ (p) ] [ without time zone ]   | TIMESTAMP                        |
+| timestamp [ (p) ] with time zone (timestamptz) | UNSUPPORTED                      |
+| tsquery                                   | STRING                                 |
+| tsvector                                  | STRING                                 |
+| txid_snapshot                             | STRING                                 |
+| uuid                                      | UUID                            |
+| xml                                       | STRING                                 |
+
+
+##  4. Scan from PostgreSQL tables with Cypher
+
+Finally, we can utilize the `LOAD FROM` statement to scan the `Person` table.
+
+```sql
+LOAD FROM uw.person
+RETURN *
+```
+
+Result:
+
+```
+┌────────┬───────┐
+│ name   │ age   │
+│ STRING │ INT64 │
+├────────┼───────┤
+│ Alice  │ 30    │
+│ Bob    │ 27    │
+│ Carol  │ 19    │
+│ Dan    │ 25    │
+└────────┴───────┘
+```
+
+##  5. Scan from PostgreSQL tables with SQL
+
+Starting from v0.9.0, Kuzu provides a `sql_query` table function allows you to execute arbitrary **read-only** 
+SQL queries on any attached Postgres database and retrieve its results for use in Kuzu. The function takes in two parameters:
+- `ATTACHED_PG_NAME`: the name of the attached postgres database to execute SQL queries.
+- `SQL_QUERY`: the read SQL query to execute.
+
+The example below shows how to scan from a SQL query which filters out all people who are younger than 20:
+```sql
+CALL SQL_QUERY('uw', 'SELECT * FROM person WHERE age > 20') 
+RETURN *
+```
+
+Result:
+
+```
+┌────────┬───────┐
+│ name   │ age   │
+│ STRING │ INT64 │
+├────────┼───────┤
+│ Alice  │ 30    │
+│ Bob    │ 27    │
+│ Dan    │ 25    │
+└────────┴───────┘
+```
+##  6. USE: Reference database without alias
+
+You can use the `USE` statement for attached databases to use a default database name for future operations.
+This can be used when reading from an attached database to avoid specifying the full database name
+as a prefix to the table name.
+
+Consider the same attached database as above:
+
+```sql
+ATTACH 'university' AS uw (dbtype postgres);
+```
+
+Instead of defining the database name for each subsequent clause like this:
+
+```sql
+LOAD FROM uw.person
+RETURN *
+```
+
+You can do:
+
+```sql
+USE uw;
+LOAD FROM person
+RETURN *
+```
+
+##  7. Copy data from PostgreSQL tables
```

Comment:
We can also COPY FROM `SQL_QUERY` I believe. Should we document it here?

Reply:
I think it's good to have an example of COPY FROM as well.